Makefile

```diff
@@ -90,25 +90,32 @@ clean: ## Clean all the temporary files
 	@rm -rf ./double-entry-generator
 	@rm -rf ./wasm-dist
 
-test: test-go test-alipay test-wechat test-huobi test-htsec test-icbc ## Run all tests
+test: test-go test-alipay-beancount test-alipay-ledger test-wechat-beancount test-wechat-ledger \
+	  test-huobi-beancount test-htsec-beancount test-icbc-beancount test-icbc-ledger ## Run all tests
```

Comment:
The comment should at the target line (I guess), or it won't print help for `test` target when using `make help`.

Reply:
Thanks for your heads-up, I wasn't aware of that before.